src/shared/selinux-util.c

```diff
@@ -210,7 +308,7 @@ void mac_selinux_maybe_reload(void) {
          * invoked since libselinux 3.2 by selinux_status_updated(3).
          * Relevant libselinux commit: https://github.com/SELinuxProject/selinux/commit/05bdc03130d741e53e1fb45a958d0a2c184be503
          * Debian Bullseye is going to ship libselinux 3.1, so stay compatible for backports. */
-        policyload = selinux_status_policyload();
+        policyload = sym_selinux_status_policyload();
```

Comment:
When initialized == LAZY_INITIALIZED, the library may not loaded yet. Hence, opening library is necessary here.